x/tokenomics/keeper/token_logic_modules.go

```diff
@@ -292,13 +292,21 @@ func (k Keeper) ProcessTokenLogicModules(
 
 	// Ensure the claim amount is within the limits set by Relay Mining.
 	// If not, update the settlement amount and emit relevant events.
-	actualSettlementCoin, err := k.ensureClaimAmountLimits(ctx, logger, &application, &supplier, claimSettlementCoin, applicationInitialStake)
+	actualSettlementCoin, err := k.ensureClaimAmountLimits(ctx, logger, &sharedParams, &application, &supplier, claimSettlementCoin, applicationInitialStake)
```

Comment:
Consider leaving a TODO about pulling this out of `Keeper#ProcessTokenLogicModules()` to come before it in `Keeper#SettlePendingClaims()`.